src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java

```diff
@@ -112,6 +138,30 @@
  * }
  * </pre>
  * <p>
+ * To configure the check to not allow {@code case, default} single-line statements without braces:
```

Comment:
I do not see benefit from this example. There is already several examples of config with tokens property overridden.
Please remove it

Reply:
done.